framework/src/Volo.Abp.Emailing/Volo/Abp/Emailing/AbpEmailingModule.cs

```diff
@@ -30,6 +31,15 @@ public override void ConfigureServices(ServiceConfigurationContext context)
                 options.FileSets.AddEmbedded<AbpEmailingModule>();
             });
 
+            Configure<AbpLocalizationOptions>(options =>
+            {
+                options.Resources
+                    .Add<EmailingResource>("en")
+                    .AddBaseTypes(
+                        typeof(EmailingResource)
```

Comment:
Why does EmailingResource inherit EmailingResource(`AddBaseTypes`)?

Reply:
thinks,I fixed it